static/js/actions/profile.js

```diff
@@ -64,6 +64,16 @@ export const updateProfileValidation = createAction(UPDATE_PROFILE_VALIDATION,
   (username, errors) => ({ errors, username })
 );
 
+export const UPDATE_VALIDATION_VISIBILITY = 'UPDATE_VALIDATION_VISIBILITY';
+export const updateValidationVisibility = createAction(UPDATE_VALIDATION_VISIBILITY,
+  (username, keySet) => ({ username, keySet })
+);
+
+export const CLEAR_VALIDATION_VISIBILITY = 'CLEAR_VALIDATION_VISIBILITY';
+export const clearValidationVisibility = createAction(CLEAR_VALIDATION_VISIBILITY,
```

Comment:
oh yeah, I wrote this but ended up not using it. `clearProfileEdit` blows the whole `edit` object away, so I'm just leaning on that.